app/src/apps/dashboard.py

```diff
@@ -42,7 +45,13 @@
         dbc.Row(
             [
                 dbc.Col(
-                    dbc.Button("Exporter les données", color="primary", className="mr-1", block=True),
+                    [
+                        html.A(
```

Comment:
Interesting way to implement the redirection using flask.
Have you read this as a good practice ?
Seems lighter than having a hidden div as I am doing everywhere else :s

Reply:
I wanted to directly use the button href attribute as you would do a redirect in standard website but somehow it didn't synchro well with my callback. Found this turnaround on the web having the button inside an html component.